build

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -x
+
+# Run unit tests and calculate code coverage.
+rm -f .coverage &&
+for dir in plsync ; do
+  PYTHONPATH=$PYTHONPATH:$PWD/site-packages coverage run --append --source ./ \
```

Comment:
Trivial question: would this line look a little cleaner by extracting the environment setting and exporting it outside the loop?